_posts/2018-10-15-12-2-Use-SkipToken-for-Server-Side-Paging.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "12.2 Use $skiptoken for server side paging "
```

Comment:
I think the new section available is 12.3?

Reply:
I did not know that you also had a PR out for a design. Updated this to be 12.3.

Reply:
There is already a 12.2 here: https://github.com/OData/WebApi/tree/gh-pages/_posts. The github pages might need to pick it up.